### ISSUE TITLE
Non-text contrast ratio between keyboard focus and background of 'Edit' button is less than required 3:1 ratio.

### ISSUE THREAD

**Issue by msftedad** [ag-grid/ag-grid#7535]
**Providing a Reproducible Scenario**
1. Open URL: https://material.angular.io/components/button/overview
2. Expand sample selection and navigate to controls
3. Verify whether the Non-text contrast ratio between keyboard focus and background of 'Edit' button is less than required 3:1 ratio or not.

**Current behavior**
Non-text contrast ratio between focus and background of 'Edit' button is 1.3:1 which is less than 3:1.

**Expected behavior**
Non-text contrast ratio between focus and background of 'Edit' button should be greater than and equal to 3:1.

**Please tell us about your environment:**
OS: Windows 11 22H2
OS Build: 23620.1000

* **AG Grid version:** X.X.X
AG Grid, version - 26.1 

* **Browser:**
Browser: Edge
Browser Version 122.0.2365.8

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
language is typescript
<img width="646" alt="218252_edit icon" src="https://github.com/ag-grid/ag-grid/assets/93735775/9804637d-0c81-4fea-a2fd-3f45f4f8b5a0">

**Comment by AG-Zoheil** [comment]
Hye @msftedad,

Looks like you were intending to raise this on here: https://github.com/angular/components?

If not and this is related to an AG Grid component you are using please update the information provided.

Kind regards,
Zoheil
